pkg/controller/machineset/machineset_controller.go

```diff
@@ -62,6 +62,8 @@ const (
 	masterProvisioningPlaybook = "playbooks/aws/openshift-cluster/provision.yml"
 
 	computeProvisioningPlaybook = "playbooks/aws/openshift-cluster/provision_nodes.yml"
+
+	deprovisioningPlaybook = "playbooks/aws/openshift-cluster/deprovision.yml"
```

Comment:
looks like the playbook name is `playbooks/aws/openshift-cluster/uninstall_nodes.yml` (from https://github.com/openshift/openshift-ansible/pull/7114). Not sure if we can tell it to deprovision one asg at a time. Will need to coordinate with @joelddiaz

Reply:
I didn't want to use an actual playbook here until the deprovision playbook work was complete.

Reply:
Cool we can update when that merges